docs/reference/tags/form_tags/create_line_item/create_line_item.md

```diff
@@ -47,13 +47,21 @@ Include a modifier
 ```
 {% endraw %}
 
-Specify adult count for a per-unit item
+Specify adult count for a per-unit item or accommodation
 {% raw %}
 ```html
   <input type="hidden" name="items[][adult_count]" value="**<adult count>**" />
 ```
 {% endraw %}
 
+Specify start date and end date for accommodation items
```

Comment:
Question; you have to have both? Can't have just a start or just an end date?

Reply:
Not for accommodations - they always need to have a checkin and checkout as they are priced by night

Reply:
Ah ok. So it's checkin/out dates not first/last nights booked (like experiences)?
So if you're booking for 1 night it's: start 1st and end 2nd, not 1st and 1st?

Reply:
Exactly, I've adjusted that phrasing now so its clearer